Resources/views/CRUD/base_acl_macro.html.twig

```diff
@@ -17,9 +17,12 @@ file that was distributed with this source code.
             {% if not admin_pool.getOption('html5_validate') %}novalidate="novalidate"{% endif %}
             >
         {% if form.vars.errors|length > 0 %}
-            <div class="sonata-ba-form-error">
-                {{ form_errors(form) }}
-            </div>
+            {% for error in form.vars.errors %}
```

Comment:
I don't know if we can, i would do, but in the symfony docs is written that the form_errors widget modify the render of all errors, fields included.